karrot/invitations/tests/test_prepare_email.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from ..factories import InvitationFactory
+from karrot.invitations.models import Invitation
+from karrot.invitations.emails import prepare_emailinvitation_email
+from karrot.utils.frontend_urls import invite_url
+
+
+class PrepareEmailTestCase(TestCase):
+    def test_prepare_email_called_with_language(self):
+        invitation: Invitation = InvitationFactory(invited_by__language='sw')
+
+        with patch('karrot.invitations.emails.prepare_email') as pe:
+            prepare_emailinvitation_email(invitation)
+            pe.assert_called_once_with(
```

Comment:
Would be better to just check the `language`, otherwise it'll start failing if we make unrelated changes to the `context` (for example).

You can get the kwargs out with something like:
```py
args, kwargs = pe.call_args
```
Then use the normal `self.assert*` methods to check the language.